Makefile

```diff
@@ -1,3 +1,12 @@
+ifeq ($(OSTYPE),darwin14)
+OS ?= Darwin
+endif
+ifeq ($(OS),Darwin) 
+sed ?= /usr/local/Cellar/gnu-sed/4.2.2/bin/sed
+else
+sed ?= sed
+endif
```

Comment:
I think that preferable to replace all this section by :
```make
SED ?= sed
```
and create your own makefile with:
```make
OS = Darwin
SED = /usr/local/Cellar/gnu-sed/4.2.2/bin/sed
include Makefile
```
Than you should do the make commands with:
`make -f <your_makefile> ...`